packages/flutter/lib/src/painting/inline_span.dart

```diff
@@ -154,12 +163,14 @@ List<InlineSpanSemanticsInformation> combineSemanticsInfo(
         );
       }
       workingLabel += effectiveLabel;
+      workingIdentifier = info.semanticsIdentifier;
     }
   }
   combined.add(
     InlineSpanSemanticsInformation(
```

Comment:
This is also some weird code, wouldn't we added am empty one if last one requiresOwnNode